terraform/modules/services/decision-tree/ecs.tf

```diff
@@ -108,6 +108,10 @@ resource "aws_ecs_task_definition" "decision_tree" {
             {
                 "name": "spring.data.neo4j.password",
                 "valueFrom": "${var.decision_tree_db_service_account_password_arn}"
+            },
+            {
+                "name": "rollbar.access.token",
+                "valueFrom": "${var.rollbar_access_token}"
```

Comment:
This variable will need adding to the this module's vars too (has to be passed down through the entire module hierarchy)